src/SprykerSdk/Zed/ComposerConstrainer/Business/ComposerConstrainerBusinessFactory.php

```diff
@@ -107,6 +106,8 @@ public function createSourceFinder(): UsedModuleFinderInterface
     }
 
     /**
+     * @deprecated Will be removed without replacement. This finder will finds all directories which are overridden on project side. We need to allow to override specific files and this finder makes it impossible to do so.
```

Comment:
With the minor (sem major) this would not be necessary.
BUT: Would be easier to release a patch and just still remove those. As no project can have used the tool yet.

Lets quickly fix all and release a patch ASAP. No BC promises necessary here still for this week.